openassessment/templates/openassessmentblock/staff_area/staff_area.html

```diff
@@ -0,0 +1,171 @@
+                                <li class="openassessment__student-info_list">
+                                    <label for="openassessment__student_username" class="label">{% trans "Single Learner Tools" %}</label>
+                                </li>
+                                <li class="openassessment__student-info_list">
+                                    <input id="openassessment__student_username" type="text" class="value" maxlength="255">
+                                </li>
+                            </ul>
+                            <ul class="list list--actions">
+                                <li class="list--actions__item">
+                                    <a aria-role="button" href="" id="submit_student_username" class="action--submit"><span class="copy">{% trans "Submit" %}</span></a>
+                                </li>
+                            </ul>
+                        </form>
+                    </div>
+                    <div id="openassessment__student-info" class="staff-info__student__report"></div>
+                </div>
+
+                {% if display_schedule_training %}
+                    <div class="staff-info__classifierset ui-staff__content__section">
+                        {% if classifierset %}
+                            <table class="staff-info__classifierset__table">
+                                <caption class="title">{% trans "Classifier set" %}</caption>
+                                <thead>
+                                <th abbr="Field" scope="col">{% trans "Field" %}</th>
+                                <th abbr="Value" scope="col">{% trans "Value" %}</th>
+                                </thead>
+                                <tbody>
+                                <tr>
+                                    <td class="value">{% trans "Created at" %}</td>
+                                    <td class="value">{{ classifierset.created_at }}</td>
+                                </tr>
+                                <tr>
+                                    <td class="value">{% trans "Algorithm ID" %}</td>
+                                    <td class="value">{{ classifierset.algorithm_id }}</td>
+                                </tr>
+                                <tr>
+                                    <td class="value">{% trans "Course ID" %}</td>
+                                    <td class="value">{{ classifierset.course_id }}</td>
+                                </tr>
+                                <tr>
+                                    <td class="value">{% trans "Item ID" %}</td>
+                                    <td class="value">{{ classifierset.item_id }}</td>
+                                </tr>
+                                </tbody>
+                            </table>
+                        {% else %}
+                            {% trans "No classifiers are available for this problem" %}
```

Comment:
@andy-armstrong  Can you help me understand what situation(s) this message shows up in?